### DISCUSSION TITLE
All testcases being skipped after installing pytest-memray

### DISCUSSION THREAD

**Discussion by ghost** [bloomberg/pytest-memray#101] [Q&A]
I have a lot of testcases that run fine. We are trying to add `memray` to our CI/CD but when we tried installing it and running the tests with `--memray` **it started skipping all the testcases!**

I tried running these with `-rsx` flag as well, this is the reason it throws for skipping the tests:
> async def function and no async plugin installed

Reminder that these test cases otherwise run fine since forever. Is there any other prerequisite for integrating `pytest-memray`?

Thanks.

**Comment by pablogsal** [comment]
Can you please provide a reproducer? 
There is nothing in the pytest plugin that requires asyncio so maybe what happened is that your pytest version got updated?

**Comment by albertvillanova** [comment]
Hola @pablogsal,

I am facing the same issue: the async tests are skipped if we pass `--memray` argument to pytest.

## Steps to reproduce the issue:
Use the following test file: `test_async.py`
```python
import pytest


@pytest.fixture
def anyio_backend():
    return 'asyncio'

@pytest.mark.anyio
async def test_async():
    assert True
```
Install required dependencies:
```shell
python -m pip install pytest anyio pytest-memray
```

The test runs as expected if `--memray` is not passed:
```shell
 python -m pytest -vv -x test_async.py
```
Output:
```
plugins: memray-1.6.0, anyio-4.0.0
collected 1 item                                                                                                                                                                                          

test_async.py::test_async PASSED
```

However, the test is skipped if we pass `--memray`:
```shell
 python -m pytest --memray -vv -x test_async.py
```
Output:
```
plugins: memray-1.6.0, anyio-4.0.0
collected 1 item                                                                                                                                                                                          

test_async.py::test_async SKIPPED (async def function and no async plugin installed (see warnings))                                                                                                 [100%]

============================================================================================ warnings summary =============================================================================================
test_async.py::test_async
  <MY_PROJECT_PATH>/.venv/lib/python3.9/site-packages/_pytest/python.py:151: PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.
  You need to install a suitable plugin for your async framework, for example:
    - anyio
    - pytest-asyncio
    - pytest-tornasync
    - pytest-trio
    - pytest-twisted
    warnings.warn(PytestUnhandledCoroutineWarning(msg.format(nodeid)))

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

**Comment by godlygeek** [comment]
This definitely seems like a bug, now that we've got a reproduction example. I've opened issue #119 for it.
